### PR TITLE
Remove EVRY

### PR DESCRIPTION
**Pull request by fbernhart**
evry

**Issue:** #3169 
**Alexa rank:** -

### ~Checklist~
~- [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)~
~- [x] I optimized the icon with SVGO or SVGOMG~
~- [x] The SVG `viewbox` is `0 0 24 24`~

### ~Description~
~I took the already used SVG.~
~I used Inkscape to fix size and center~
~I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG~

### Edit
This PR is removing the EVRY icon as the brand no longer exists.

### CODE REVIEW

File: icons/evry.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...154-2.832-.066-3.42.034-.227.025-.495.75-.617zm8.845.065c.302.16-.045.855-.626 1.277-.374.272-1.014.679-1.672.679a6.206 6.206 0 011.017-1.298c.47-.461 1.018-.8 1.281-.658zm8.17-.153c.29-.373.349-.596-.133-1.061-.433-.417-.723-.171-.934.045-.035.042-1.016 1-2.049 2.17-.804.91-1.499 1.756-1.499 1.756s-.025-.57.062-1.286c.049-.408.186-1.201.226-1.453.204-1.274-.593-1.325-.89-1.343-.312-.02-.46.145-.554.533-.085.356-.55 2.31-.365 4.325.033.363.105.737.112.76l-.188.258c-1.749-.485-2.946-1.664-2.946-1.664s.574-.165 1.204-.543c.501-.3 1.465-1.043 1.363-2.353-.087-1.112-1.27-1.62-1.707-1.53-.397.084-1.09.559-1.747 1.14-.044-.303-.052-.638-.368-.675-.232-.027-.5.082-.696.182a1.116 1.116 0 00-.5.473l-.737 1.665c-.024.059-.072.072-.11.08-.394.09-.52.37-.549.449 0 0-.165.347.126.66-.146.386-.533 1.283-.533 1.283-.48 1.1-.449 1.483-.277 1.717.087.119.393.29.762.35.278.047.491-.118.723-.481.343-.537 1.032-1.953 1.032-1.953s1.37 1.474 4.121 2.443c0 0-1.455 1.973-1.665 3.064 0 0-.108.715.872.713.246 0 .617-.058.886-.221a.475.475 0 00.223-.281.216.216 0 00-.02-.153c-.028-.047-.07-.073-.102-.104-.05-.05-.043-.113-.016-.187.54-1.544 2.732-4.17 3.197-4.738 1.233-1.507 3.512-3.854 3.673-4.041z"/></svg>
```
Comment (adamrusted) [inline icons/evry.svg:1]: Looks like some differences between PR (red) and source (black) - would you mind rebuilding this one?
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by fbernhart** [comment]
Seems like **EVRY** alone doesn't even exist anymore. They were bought by **Tieto** and the company is called **TietoEVRY** now. https://www.tietoevry.com/en/newsroom/all-news-and-releases/stock-exchange-releases/2019/12/merger-between-tieto-and-evry-completed--tietoevry-established/

They as well have a combined logo now, as can be seen [in the header here](https://www.tietoevry.com).


@simple-icons/maintainers how do we go about this one? Renaming a brand is a breaking change, right?

How about just updating the "EVRY" icon in this PR and then creating a new PR, marked as breaking change, to rename it?

**Comment by ericcornelissen** [comment]
> @simple-icons/maintainers how do we go about this one? Renaming a brand is a breaking change, right?
> 
> How about just updating the "EVRY" icon in this PR and then creating a new PR, marked as breaking change, to rename it?

As renaming (=removing EVRY and adding TietoEVRY) is indeed a breaking change we cannot do that now. We could have both available for the duration of v4 and remove EVRY with [the v5 release](https://github.com/simple-icons/simple-icons/milestone/4), or could the rename only with the v5 release. Either are acceptable to me: having two icons for "one" brand is a bit weird, but having the Tieto-version available before v5 may help users switch before that.

**Comment by adamrusted** [comment]
> but having the Tieto-version available before v5 may help users switch before that.

Agree with this, may be better for us to give users a decent run-up to the deprecation of the old icon.

**Comment by mondeja** [comment]
Could you remove it using this pull and mark it as breaking change for the next v5 release @fbernhart?

**Comment by PeterShaggyNoble** [comment]
@fbernhart, could you update the original post to clarify that this PR will remove the icon, please?

**Review by PeterShaggyNoble — APPROVED** [review]
Looks good to me 👍🏻 Once the conflicts have been resolved, this can be merged straight in.

**Pull request merged**